doc/nrf/app_dev/device_guides/nrf53/qspi_xip_guide_nrf5340.rst

```diff
@@ -205,8 +205,7 @@ Programming the project
 For the nRF5340 DK and other boards equipped with flash working in the QSPI mode, use the :ref:`standard programming command <programming>` (``west flash``).
 For other cases, set up a configuration file for nrfjprog, as described in the following section.
 
-.. note::
-      |nrfjprog_deprecation_note|
+##need edits
```

Comment:
@nordicjm @carlescufi Hi, could you provide inputs on how to do the same without using nrfjprog? We are deprecating this in ncs 3.0

Reply:
this is outdated, rebase